tests/Console/Output/NullOutputTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Console\Output;
+
+use PhpCsFixer\Console\Output\NullOutput;
+use PhpCsFixer\Tests\TestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Console\Output\NullOutput
+ */
+final class NullOutputTest extends TestCase
+{
+    public function testNullOutput()
+    {
+        $output = new NullOutput();
+        $output->printLegend();
+        // no exceptions, no output, so OK
+        $this->addToAssertionCount(1);
```

Comment:
oh my... looks extra hacky to bypass "no assertion in test" warnign

Reply:
no, there is no `assertNoExceptionHasBeenThrown` and/or `assertNoEchoOrPrintHasBeenDoneToStdOutOrStdErr` AFAIK

Reply:
maybe raise a concern on phpunit ?
either we will know the official way, or it will be a feature request

Reply:
Maybe https://github.com/sebastianbergmann/phpunit/issues/2484#issuecomment-278233214

Reply:
`@doesNotPerformAssertions` seems to effect coverage :|

Reply:
@SpacePossum , when I use the annotation, I see the coverage:
https://github.com/sebastianbergmann/phpunit/issues/2484#issuecomment-357230090

Reply:
updated to use the annotation :+1: